packages/ripple-nuxt-tide/README.md

```diff
@@ -2,7 +2,180 @@
 
 > The Nuxt module for adding Tide integration.
 
-## Setup
+## Nuxt configuration
+
+A `nuxt.config.js` file should exist in the root of your nuxt-tide site.
+
+The basic anatomy of a `nuxt.config.js` file is:
+
+```js
+export default {
+  modules: [],
+  debug: false,
+  serverMiddleware: {},
+  head: {},
+  loading: {},
+  router: {},
+  styleResources: {},
+  ripple: {},
+  tide: {}
+}
+```
+
+### modules
+
+An array of all modules to include.
+See [Nuxt configuration modules](https://nuxtjs.org/api/configuration-modules).
+
+```js
+{
+  modules: [
+    '@dpc-sdp/ripple-nuxt-tide',
+    '@nuxtjs/style-resources',
+    // we want to always disallow bots on the example site
+    ['@nuxtjs/robots', {
+      UserAgent: '*',
+      Disallow: '/'
+    }]
+  ],
+}
+```
+
+### debug
```

Comment:
nuxt doc as well. Shouldn't be included.

Reply:
Removed.